src/easylink/configuration.py

```diff
@@ -125,14 +178,26 @@ def spark_resources(self) -> dict[str, Any]:
     #################
 
     def _get_schema(self, potential_schemas: list[PipelineSchema]) -> PipelineSchema:
-        """Validates the requested pipeline against supported schemas.
+        """Returns the first pipeline schema that successfully validates the requested pipeline.
```

Comment:
Because I turned on documenting of private members, we should take more care to document them (at least the important ones like this).